packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -279,42 +315,54 @@ export class ChildLogger extends TelemetryLogger {
                 if(extendedProps !== undefined) {
                     if(extendedProps.all !== undefined) {
                         combinedProperties.all = {
-                            ... combinedProperties.all,
-                            ... extendedProps.all,
+                            ...combinedProperties.all,
+                            ...extendedProps.all,
                         };
                     }
                     if(extendedProps.error !== undefined) {
                         combinedProperties.error = {
-                            ... combinedProperties.error,
-                            ... extendedProps.error,
+                            ...combinedProperties.error,
+                            ...extendedProps.error,
                         };
                     }
                 }
             }
 
+            // Allow global properties to be updated over time
+            Object.assign(baseLogger.globalProperties, globalProperties);
+
+            // Concatenate next layer version to existing versions
+            if (namedLayerVersion !== undefined) {
```

Comment:
i really don't like this. it'll be hard to aggregate. i prefer the version per column approach today

Reply:
at a minimum this should be json, which has support for parsing in most data tools

Reply:
Yeah, I'm good to ditch it.